app/migrations.go

```diff
@@ -156,3 +157,58 @@ func (a *App) DoEmojisPermissionsMigration() {
 		mlog.Critical(fmt.Sprint(result.Err))
 	}
 }
+
+func (a *App) DoGuestRolesCreationMigration() {
```

Comment:
Also, where do we add the missing Guest roles to custom schemes that already exist?

Reply:
Nope, I going to fix that

Reply:
Done!